tests/GraphQL.Integration.Tests/Helpers/WebHostHelpers.cs

```diff
@@ -41,9 +41,9 @@ public static GraphQLHttpClient GetGraphQLClient(int port, string endpoint, bool
             => new GraphQLHttpClient(new GraphQLHttpClientOptions
             {
                 EndPoint = new Uri($"http://localhost:{port}{endpoint}"),
-                UseWebSocketForQueriesAndMutations = requestsViaWebsocket,
-                JsonSerializer = serializer ?? new NewtonsoftJsonSerializer()
-            });
+                UseWebSocketForQueriesAndMutations = requestsViaWebsocket
+            },
+                serializer ?? new NewtonsoftJsonSerializer());
```

Comment:
Do you have tests for STJ serializer?

Reply:
yes, there is always an abstract base class containing the tests and a specific testclass for each serializer (currently Newtonsoft and `System.Text.Json`)